Unity.Mathematics.TestProject/Assets/Scenes/HashAligned.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Unity.Burst;
+using System;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Mathematics;
+
+[BurstCompile]
```

Comment:
Any reason why a new monobehaviour needs to be added and added to the sample scene? I'm not familiar with math's test setup so maybe this is normal, but I'd figure validation of hash correctness on players would be handled by the standalone testing for the `[Test]`s you added

Reply:
It was to test with a player build. The mathematics CI actually doesn't build any players so we have a huge hole in testing there. How do I build all the tests to run in a player?

Reply:
The test asmdef is currently specified to only run in editor:
```
    "includePlatforms": [
        "Editor"
    ],
```
https://github.com/Unity-Technologies/Unity.Mathematics/blob/master/src/Tests/Unity.Mathematics.Tests.asmdef#L10

If you change it to not specify the "Editor" paltform it will default to all platforms. e.g.

```
{
    "name": "Unity.Mathematics.Tests",
    "rootNamespace": "",
    "references": [
        "Unity.Mathematics",
        "UnityEngine.TestRunner",
        "UnityEditor.TestRunner"
    ],
    "includePlatforms": [],
    "excludePlatforms": [],
    "allowUnsafeCode": true,
    "overrideReferences": true,
    "precompiledReferences": [
        "nunit.framework.dll"
    ],
    "autoReferenced": false,
    "defineConstraints": [
        "UNITY_INCLUDE_TESTS"
    ],
    "versionDefines": [],
    "noEngineReferences": false
}
```